lib/process-overrides.js

```diff
@@ -0,0 +1,13 @@
+module.exports = processOverrides;
+function processOverrides(urls, scenarios) {
+
+    scenarios.forEach((scenario, scenarioKey) => {
```

Comment:
I know that doing a forEach inside another forEach isn't ideal, but I haven't yet figured out a more efficient way to do this.